parse_solver_output.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+# workflow-helpers
+# Copyright(C) 2020 Francesco Murdaca, Bissenbay Dauletbayev
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""This script run in a workflow task to parse solver output and produce inputs for Kafka message."""
+
+import os
+import logging
+import json
+
+from typing import List
+from thoth.storages import GraphDatabase
+from thoth.storages import AdvisersResultsStore
+from thoth.storages.graph.enums import ThothAdviserIntegrationEnum
+
+from thoth.workflow_helpers.common import retrieve_solver_document
+from thoth.workflow_helpers import __service_version__
+
+GRAPH = GraphDatabase()
+GRAPH.connect()
+
+ADVISER_STORE = AdvisersResultsStore()
+ADVISER_STORE.connect()
+
+component_name = os.environ["THOTH_MESSAGING_COMPONENT_NAME"]
+document_path = os.environ["THOTH_SOLVER_DOCUMENT_PATH"]
+
+_LOGGER = logging.getLogger("thoth.parse_solver_output")
+_LOGGER.info("Thoth workflow-helpers task: parse_solver_output v%s", __service_version__)
+
+
+def _check_unsolved_packages(
+    unsolved_packages: List[str], package_name: str, package_version: str, index_url: str
```

Comment:
In original https://github.com/thoth-station/workflow-helpers/blob/master/parse_solved_package.py#L46, we have `indexes`, but as we are getting just `index_url` https://github.com/thoth-station/workflow-helpers/pull/107/files#diff-9295e229a712f115d192ebe84a0d7f73R86, I decided to make it single `index_url` instead of `indexes`, so WDYT @pacospace ?

Reply:
Sounds great! Thanks :100: